### PR TITLE
Add a Pull Request Template

### PR DESCRIPTION
**Pull request by flawedmatrix**
## Description
This PR adds a pull request template so that the process of creating new pull requests can be more formalized. This pull request template also provides a helpful reminder to check out the Contribution Workflow.

## Related Issues
Updates #13 .

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+name: Pull request
+about: Tell us about your contribution
+---
+
+<!-- Thanks for contributing to Cross-cluster-connectivity! -->
```
Comment (mcwumbly) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_template.md:6]: nit: I think we've esablished the name as "Cross-cluster Connectivity" per the opening line of the README. Let's use that for consistency (and do a find/replace all later if we update our capitalization/dasherization preferences)

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+-->
+
+## Description
+<!-- A clear and concise description of what the PR is adding or changing. -->
+
+## Related Issues
+
+<!--
+All PR's should have a `Fixes #NNN` or `Updates #NNN` line in the pull request
+description. Cross-cluster-connectivity operates according to the talk, then
```
Comment (mcwumbly) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_template.md:20]: same here.

**Review by mcwumbly — CHANGES_REQUESTED** [review]

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md (hunk 1 above)
Comment (christianang) [reply .github/PULL_REQUEST_TEMPLATE/pull_request_template.md]: Good catch, that was unintentional.

**Review by christianang — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+---
+name: Pull request
+about: Tell us about your contribution
+---
+
+<!-- Thanks for contributing to Cross-cluster connectivity! -->
```
Comment (mcwumbly) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_template.md:6]: nit: s/connectivity/Connectivity/

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+-->
+
+## Description
+<!-- A clear and concise description of what the PR is adding or changing. -->
+
+## Related Issues
+
+<!--
+All PR's should have a `Fixes #NNN` or `Updates #NNN` line in the pull request
+description. Cross-cluster connectivity operates according to the talk, then
```
Comment (mcwumbly) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_template.md:20]: nit: s/connectivity/Connectivity/

**Review by mcwumbly — CHANGES_REQUESTED** [review]

**Review by mcwumbly — APPROVED** [review]

**Pull request merged**